File: easy/Q1/reverse.js
// Function to reverse the value
const reverse = (stringValue) => stringValue.split("").reverse().join("");

// Function to check type of the value
const checkType = (value) =>
  typeof value === "string" ? reverse(value) : reverse(value.toString());

console.log(checkType("Hello, World")); // dlroW ,olleH
console.log(checkType(1234)); // 4321

File: easy/Q1/reverse.py
def reverse(value):
    ''' Function to reverse the value after checking the datatype of {value} '''
    return value[::-1] if type(value) == str else str(value)[::-1]


if __name__ == "__main__":
    print(reverse("Hello, World"))  # dlroW ,olleH
    print(reverse(1234))  # 4321
